group-owner.html

```diff
@@ -262,7 +287,13 @@
 
 <p>Any data imported by a user will be owned by them and will be in a group of which they are a member. For details on importing see <a href="http://help.openmicroscopy.org/importing-data-5.html" target="_blank" class="external-url">Importing Data</a>.
 
-<p>Group Owners and Administrators, who are members of the group, can import data for any user in the group i.e. the data will belong to that user.</p>
+<p>
+    Group Owners can import data for other users in their group i.e. the data
+    will belong to that user. Administrators can also perform this action but
+    they must be members of the target group to do so using OMERO.insight or
+    OMERO.importer (OMERO.cli will allow admins to
+    <a href="https://docs.openmicroscopy.org/latest/omero/users/cli/import.html#advanced-import-commands">'import for'</a>
```

Comment:
Resolves as ``...admins to import for without being....`` -> replace with ``...admins to import for others without being...`` (add ``others`` to keep the grammatical structure better understandable)